Doc/library/imaplib.rst

```diff
@@ -204,7 +204,15 @@ An :class:`IMAP4` instance has the following methods:
 
 .. method:: IMAP4.append(mailbox, flags, date_time, message)
 
-   Append *message* to named mailbox.
+   Append *message* to a specified mailbox.
+   
+   *mailbox* ( ``string``): name of the mailbox where the message should be appended
+   
+   *flags* (list of ``string`` ): The flags parameter is a sequence of flags associated with the message. Flags are used to mark messages with certain attributes or states. Common flags include ``\\Seen`` (marks the message as read), ``\\FlagName`` (custom flags),  as (``flags = ["\\Seen", "\\FlagName]``) etc.
+
+   *date_time* (``string``): date and time associated with the message. Often of the format *"DD-Mon-YYYY HH:MM:SS" (e.g., "01-Jan-2023 12:00:00")*
```

Comment:
Remove ``string`` and [wrap to 80 characters](https://devguide.python.org/documentation/style-guide/#use-of-whitespace):
```suggestion
   *date_time* is the date and time associated with the message.
   Often of the format *"DD-Mon-YYYY HH:MM:SS" (e.g., "01-Jan-2023 12:00:00")*.
```

I'm curious, where did you get that it's often of this format?